src/palindrom.js

```diff
@@ -780,6 +769,16 @@ var Palindrom = (function() {
         );
       }
     });
+    /* when patches are applied, and their `value`s are objects, make sure to pass the proxified version */
+    sequence
+      .filter(
+        el => ['replace', 'add'].includes(el.op) && typeof el.value === 'object'
+      )
+      .map(patch => {
+        const newPatch = { op: '_get', path: patch.path };
+        jsonpatch.apply(this.obj, [newPatch]);
+        patch.value = newPatch.value;
+      });
```

Comment:
I guess yes we do. Consider the following model is set in the first `callback` callback of Palindrom:
```js
model = {person: {name: 'Omar'}}
```
Then the following patch is applied:
```js
{op: 'replace', path: '/person': value: {name: 'Tomek'}}
```
It will be applied to the `model` inside Palindrom quite well. But then `patch-applied` is fired to `palindrom-client` with a patch of a value `{name: 'Tomek'}` that doesn't equal `model.person`. They have different references and this breaks Polymer.